met-web/src/utils/index.ts

```diff
@@ -47,3 +47,16 @@ export const blobToFile = (theBlob: Blob, fileName: string): File => {
     //Cast to a File() type
     return <File>theBlob;
 };
+
+export const determinePathSegments = () => {
+    const url = new URL(window.location.href);
+    // filters out empty segments, which can occur if there are leading or trailing slashes in the pathname
+    const pathSegments = url.pathname.split('/').filter((segment) => segment.trim() !== '');
+    return pathSegments;
+};
+
+export const determineBaseName = () => {
```

Comment:
I think the name of this function is potentially a bit confusing. When I think of a URL's "base" I would think of the domain name itself. Here it looks like this function is meant to find the tenant code at the beginning of the URL path?

Maybe you could change the comment to explain the function a bit more?

Reply:
Make sense! Updated